addons/binding/org.openhab.binding.rme/OSGI-INF/RMEHandlerFactory.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014 openHAB UG (haftungsbeschraenkt) and others.
```

Comment:
This causes build failures whatever the directory I attempt to run it from.

[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.800 s
[INFO] Finished at: 2016-02-28T17:53:21+01:00
[INFO] Final Memory: 5M/81M
[INFO] ------------------------------------------------------------------------
[ERROR] Could not find goal 'format' in plugin org.codehaus.mojo:license-maven-plugin:1.8 among available goals third-party-report, check-file-header, add-third-party, update-project-license, help, aggregate-add-third-party, license-list, comment-style-list, update-file-header, download-licenses -> [Help 1]